code/go/0chain.net/smartcontract/storagesc/allocation_eventdb.go

```diff
@@ -12,7 +12,12 @@ import (
 	"0chain.net/smartcontract/stakepool"
 )
 
-func allocationTableToStorageAllocation(alloc *event.Allocation, eventDb *event.EventDb) (*StorageAllocation, error) {
+type AllocationData struct {
+	StorageAllocation `json:",inline"`
+	Blobbers          []*StorageNode
```

Comment:
Add json tag `json:"blobbers`, otherwise the API will return different json format.